.github/workflows/update-helm.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     name: update helm chart
     runs-on: ubuntu-latest
     steps:
+      # TODO: check for new docker images via docker hub repo.
```

Comment:
this also needs to check if a new container update is available and only update the hel chart then.

Reply:
done